Sources/Utility/Process.swift

```diff
@@ -179,7 +179,10 @@ public final class Process: ObjectIdentifierProtocol {
 
         // Set the attribute flags.
         var flags = POSIX_SPAWN_SETSIGMASK | POSIX_SPAWN_SETSIGDEF
-        flags |= POSIX_SPAWN_SETPGROUP
+        // To allow taking input from terminal, only set this flag when stdin is not a tty.
+        if !isatty(STDIN_FILENO) {
+           flags |= POSIX_SPAWN_SETPGROUP
+        }
```

Comment:
Can you confirm if it is a real issue? I would expect we fail to actually kill subprocesses spawned this way, since we try and signal a non-existent process group.